features/adressering/aanhef/overzicht.feature

```diff
@@ -0,0 +1,96 @@
+# language: nl
+
+@info-api
+Functionaliteit: leveren van de aanhef van een persoon
+  Als consumer
+  wil ik de aanhef (introducerende zin van een brief of e-matching) van een persoon kunnen vragen
+  zodat ik de persoon op de juiste en consistente manier kan aanschrijven
+
+  De volgende afkortingen worden gebruikt in deze specificatie:
+  - GA = "Geachte mevrouw", "Geachte heer" of "Geachte VL" afhankelijk van het geslacht
+  - VL = voorletters
+  - VV = voorvoegsel.
+  - GN = geslachtsnaam
+  - VP = voorvoegsel partner
+  - GP = geslachtsnaam partner
+
+  Regel: de aanhef van een persoon zonder partner wordt als volgt samengesteld: GA VV GN. Wanneer een persoon geen voorvoegsel heeft, wordt de overbodige spatie niet opgenomen
+
+    Abstract Scenario: persoon met voornamen, geslachtsnaam en geen voorvoegsel heeft geslacht <voorbeeld>
+      Gegeven de persoon 'Jo' met burgerservicenummer '000000141'
+      * heeft de volgende gegevens
+        | naam                        | waarde     |
+        | geslachtsaanduiding (04.10) | <geslacht> |
+        | voornamen (02.10)           | Jo Rene    |
+        | geslachtsnaam (02.40)       | Groenen    |
+      Als 'adressering.aanhef' wordt gevraagd van personen gezocht met burgerservicenummer van 'Jo'
+      Dan wordt 'Jo' gevonden
```

Comment:
deze stap doet niks, toch? (er is geen burgerservicenummer gevraagd) is die toegevoegd voor de leesbaarhied?

Reply:
implementatie aangepast zodat deze stap niet meer noodzakelijk is